Source Code/Ticket Management System/customer_menu.c

```diff
@@ -87,10 +87,22 @@ void listTickets()
 void viewFAQ()
 {
 	wprintf(L"Q: My computer is on but I can't see anything on the screen.\n");
-	wprintf(L"A: Try moving the mouse or hitting any key on the keyboard.\nIf that fails, locate the power button on the screen and press it.\n");
+	wprintf(L"A: Try moving the mouse or hitting any key on the keyboard.\nIf that fails, locate the power button on the screen and press it/ncheck the connectivity of the screen monitor to the computer, is and check if the screen is on.\n");
 	wprintf(L"\n");
 	wprintf(L"Q: I sent a document to print but it doesn't do anything\n");
-	wprintf(L"A: Check if the printer has enough paper. Check if the paper tray is fully engaged. If all else fails, contact technical support.\n");
+	wprintf(L"A: Check if the printer has enough paper. Check if the paper tray is fully engaged. check for paper jams. If all else fails, contact technical support.\n");
+	wprintf(L"\n");
+	wprintf(L"Q: I can't turn on my computer\n");
+	wprintf(L"A: first of all check the electric supply, and if the switch behind the computer is on, after that try to disconnect the cable cord from the computer, and keep the power button pressed 10 seconds,/n then reinsert the power cord and try to turn on the computer.\n");
+	wprintf(L"\n");
+	wprintf(L"Q: My network doesn't work(there's an X symbol over the internet symbol)\n");
+	wprintf(L"A: Try to check if there is light comming up from the ethernet cable (it's on the lower side behind the computer) check connectivity of said cable, right click that icon, make sure the internet isn't dissabled, if using netstick try to reinset the netstick.\n");
```

Comment:
`comming` - spelling error. Use `coming`.

`dissabled` - spelling error. Use `disabled`.

`try to reinset the netstick` - spelling error. Use `reinsert`.